java-rx-smgrsdk/src/main/java/io/vantiq/svcconnector/InstanceConfigUtils.java

```diff
@@ -43,9 +43,9 @@ public Properties loadServerConfig() {
 
     public Properties loadServerSecrets(String fileName) {
         Properties secrets = new Properties();
-        val resourceUrl = Resources.getResource(fileName);
         ByteArrayInputStream propertyStream = null;
         try {
+            val resourceUrl = Resources.getResource(fileName);
             propertyStream = new ByteArrayInputStream(Resources.toByteArray(resourceUrl));
             secrets.load(propertyStream);
         } catch (IOException ignored) {
```

Comment:
also need to catch IllegalArgumentException here, since that is what Resource.getResource() returns if it does not find the resource.    (seems odd for that to be the exception returned, but....)
```suggestion
        } catch (IllegalArgumentException ignored) {
            // Nothing found when calling getResource, continue
        } catch (IOException ignored) {
```